src/RefractAST.h

```diff
@@ -2,27 +2,19 @@
 //  RefractAST.h
 //  drafter
 //
-//  Created by Jiri Kratochvil on 18/05/15.
+//  Created by Jiri Kratochvil on 31/07/15.
```

Comment:
We generally use creation dates in C++ projects. I don't mind them since Xcode automatically makes them. This is a case where the date was wrong since Jirka copied it from another file and I forgot to comment on this in the previous PR.